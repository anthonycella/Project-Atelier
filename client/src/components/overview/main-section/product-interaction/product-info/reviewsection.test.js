import renderer from 'react-test-renderer';
import ReviewSection from './reviewsection.jsx';

it('Review Section component renders to the page', () => {
  const component = renderer.create(
    <ReviewSection />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
