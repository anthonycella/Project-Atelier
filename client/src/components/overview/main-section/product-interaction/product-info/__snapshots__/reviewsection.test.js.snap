// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review Section component renders to the page 1`] = `
<div>
  Review
  <a
    href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
  >
    Read all reviews
  </a>
</div>
`;
